tutorminio/patches/openedx-auth

```diff
@@ -1,3 +1,4 @@
 AWS_STORAGE_BUCKET_NAME: "{{ MINIO_BUCKET_NAME }}"
 AWS_S3_CUSTOM_DOMAIN: ""
-AWS_QUERYSTRING_AUTH: false
\ No newline at end of file
+AWS_QUERYSTRING_AUTH: {% if MINIO_QUERYSTRING_AUTH %}true{% else %}false{% endif %}
+AWS_S3_SECURE_URLS: {{ ENABLE_HTTPS }}
```

Comment:
This is going to be rendered as:

```
AWS_S3_SECURE_URLS: True
```

or:

```
AWS_S3_SECURE_URLS: False
```

But "True" and "False" (with uppercase) are not yaml booleans. They are strings. Which will mean that `AWS_S3_SECURE_URLS` will always evaluate to "true".